### ISSUE TITLE
Comments or bold text in table break syntax highlight

### ISSUE THREAD

**Issue by kazerniel** [Frederisk/Wikitext-VSCode-Extension#98]
![image](https://github.com/user-attachments/assets/8175d8d2-45cf-41b3-bf33-bb2302b0c1d4)

<details>
<summary>the sample code I used</summary>

```wiki
{|
! style="padding: 0.5rem 1.5rem;" | Title 1
! style="padding: 0.5rem 1.5rem;" | Title 2
! style="padding: 0.5rem 1.5rem;" | Title 3
! style="padding: 0.5rem 1.5rem;" | Title 4
! style="padding: 0.5rem 0.5rem;" | Title 5
! style="padding: 0.5rem 1.5rem;" | Title 6
! style="padding: 0.5rem 1.5rem;" | Title 7
|- style="text-align: center; background: linear-gradient(#d7e1cc, #ccd9bf);" <!-- comment -->
| style="padding: 0.5rem 1.5rem;" | '''bold text'''
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | text
| style="padding: 0.5rem 0.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}
|- style="text-align: center; background: linear-gradient(#e2dbf1, #dbd2ed);"
| style="padding: 0.5rem 1.5rem;" | '''bold text'''
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | text
| style="padding: 0.5rem 0.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}
|- style="text-align: center; background: linear-gradient(#e5e9f1, #dee3ed);"
| style="padding: 0.5rem 1.5rem;" | text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | text
| style="padding: 0.5rem 0.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}<br />text
| style="padding: 0.5rem 1.5rem;" | {{template}}
|}
```
</details>

**Comment by Frederisk** [comment]
This complex nested syntax is a bit difficult to maintain using tmLanguage, I will spend some time improving them.

**Comment by Frederisk** [comment]
Fixed in version 3.8.3.
